war/src/main/less/base/style.less

```diff
@@ -835,8 +835,8 @@ table.parameters > tbody:hover {
     opacity: 0.5;
 }
 #buildHistoryPage .build-search-row .clear:hover {
-    background-color: #d3d3d3;
-    border: 1px solid #bbb;
+    background-color: var(--very-light-grey);
+    border: 1px solid var(--very-light-grey);
```

Comment:
Is this effective removal of the border deliberate?

Reply:
Mistake, I'm changing it up

Reply:
It wasn't, I'm setting it to `--medium-grey`, the color normally used for borders.